bindings/ios/MEGASdk.h

```diff
@@ -5400,6 +5401,32 @@ typedef NS_ENUM(NSInteger, AffiliateType) {
  */
 - (void)killSession:(uint64_t)sessionHandle;
 
+/**
+ * @brief Returns the deadline to remedy the storage overquota situation
+ *
+ * This value is valid only when [MegaSdk getUserData] has been called after
+ * receiving a callback [MegaDelegate/MegaGlobalDelegate onEvent:event] of type
```

Comment:
MegaGlobalDelegate is enough here, we don't need to expose sdk class names or interfaces